src/views/Timeline.vue

```diff
@@ -64,15 +62,6 @@
         dense
         class="pa-0 ma-0"
       >
-        <infinite-loading
```

Comment:
@vickywane Why did you change the semantics of the top and bottom spinner. I don't think you preserved the behavior. Please keep PR scoped to a minimal possible change. This PR should only focus on the PeerID visibility. If you want to suggest change of the top/bottom infinite loader and spinners, please open a separate issue to discuss.

Reply:
This came in from rebasing. Not an intentional adjustment.

I just restored this file using the copy in the master branch. Is that the latest released version?